docs/tools/vdb_table/data/singlestoredb.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "SingleStoreDB",
+  "links": {
+    "docs": "https://docs.singlestore.com/",
+    "github": "https://github.com/singlestore-labs",
+    "website": "https://www.singlestore.com/",
+    "vendor_discussion": "",
+    "poc_github": "",
+    "slug": "singlestoredb"
+  },
+  "oss": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "",
+    "comment": ""
+  },
+  "dev_languages": {
+    "value": [
+      "c++",
+      "go"
+    ],
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_launch_year": 2017,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-vector-data/",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-vector-data/",
+    "comment": ""
+  },
+  "facets": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Supported through SQL aggregations"
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/reference/sql-reference/data-types/geospatial-types/",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Multiple vector columns can be created and queried"
```

Comment:
Multi-vector means that the query contains multiple vectors and the results get scored by their proximity to the closest query vector. Is this what you had in mind here? Any way to check documentation for this?

Reply:
No direct documentation for this but this is possible:

```
SELECT 
    id,
    name,
    GREATEST(
        vector_embedding <*> @query_vector1,
        vector_embedding <*> @query_vector2,
        vector_embedding <*> @query_vector3
    ) AS best_similarity
FROM 
    your_table
ORDER BY 
    best_similarity DESC
```